py_nltk/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM socrata/py_analysis
+
+RUN pip install nltk==3.0.0
+
+RUN mkdir -p /home/socrata
+
+ENV HOME /home/socrata
+
+RUN python -m nltk.downloader -d $HOME/nltk_data stopwords &&\
+    python -m nltk.downloader -d $HOME/nltk_data punkt &&\
+    python -m nltk.downloader -d $HOME/nltk_data wordnet &&\
+    python -m nltk.downloader -d $HOME/nltk_data maxent_treebank_pos_tagger
```

Comment:
noah, good question.    I thought that pulling down nltk 3.0.0  and using it's download script would bring a particular version,  but looking at nltk's downloader.py script, there are these comments followed by the more-or-less obvious code:

```python
# Do we already have the current version?
# Check for (and remove) any old/stale version.
```

But I don't see anything (in nltk's library or on the web) that would suggest we can specify a version.  There is an argument to the downloader that allows for svn versioning, but we don't use svn.  

Thoughts?